Description: Work around LP: #546578 by disabling the fade effect on screen locking
Bug-Ubuntu: https://launchpad.net/bugs/546578
Forwarded: not-needed
Author: Chris Coulson <chris.coulson@canonical.com>

diff -Nur -x '*.orig' -x '*~' gnome-screensaver-2.30.0/src/gs-manager.c gnome-screensaver-2.30.0.new/src/gs-manager.c
--- gnome-screensaver-2.30.0/src/gs-manager.c	2010-05-17 15:10:54.313445797 +0100
+++ gnome-screensaver-2.30.0.new/src/gs-manager.c	2010-05-17 15:33:41.393445141 +0100
@@ -1754,7 +1754,7 @@
         manager->priv->active = TRUE;
 
         /* fade to black and show windows */
-        do_fade = TRUE;
+        do_fade = FALSE;
         if (do_fade) {
                 manager->priv->fading = TRUE;
                 gs_debug ("fading out");
